examples/DocQA/docker/llama_stack_start.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# Sleep for 45 seconds
+sleep 45
```

Comment:
why is this needed?!

Reply:
Ollama needs to download the model for the first time and llama-stack will produce ollama can not found if we do not wait for ollama to be complete. I borrow the idea from [here](https://github.com/meta-llama/llama-stack/blob/435f34b05e84f1747b28570234f25878cf0b31c4/distributions/dell-tgi/compose.yaml#L45)

Reply:
ok we should add a comment there saying this is a wait for the model to get loaded

Reply:
sure, will add this